main.go

```diff
@@ -37,14 +37,18 @@ func main() {
 			log.Fatal(err)
 		} else {
 			os.Setenv("REGISTRY_STORAGE_S3_ACCESSKEY", string(accesskey))
```

Comment:
I have a feeling that while everything below will work, object-storage-cli will not pick up on IAM credentials. Have you tested this and registry blobs are being uploaded to S3 as expected?